docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,281 @@
+[API convention](https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md).
+
+The phase of a Pod is a simple, high-level summary of where the Pod is in its
+lifecycle. The phase is not intended to be a comprehensive rollup of observations
+of Container or Pod state, nor is it intended to be a comprehensive state machine.
+
+The number and meanings of Pod phase values are tightly guarded.
+Other than what is documented here, nothing should be assumed about Pods that
+have a given `phase` value.
+
+Here are the possible values for `phase`:
+
+* Pending: The Pod has been accepted by the Kubernetes system, but one or more of
+  the Container images has not been created. This includes time before being
+  scheduled as well as time spent downloading images over the network,
+  which could take a while.
+
+* Running: The Pod has been bound to a node, and all of the Containers have been
+  created. At least one Container is still running, or is in the process of
+  starting or restarting.
+
+* Succeeded: All Containers in the Pod have terminated in success, and will not
+  be restarted.
+
+* Failed: All Containers in the Pod have terminated, and at least one Container
+  has terminated in failure. That is, the Container either exited with non-zero
+  status or was terminated by the system.
+
+* Unknown: For some reason the state of the Pod could not be obtained, typically
+  due to an error in communicating with the host of the Pod.
+
+## Pod conditions
+
+If any Container in a Pod specifies a readiness probe, the Pod reports a
+[PodCondition](/docs/resources-reference/v1.5/#podcondition-v1).
+A PodCondition has a `status` field that can be True, False, or Unknown.
+
+## Container probes
+
+A [Probe](/docs/resources-reference/v1.5/#probe-v1) is a diagnostic
+performed periodically by the kubelet on a Container. To perform a diagnostic,
+the kublet calls a
+[Handler](https://godoc.org/k8s.io/kubernetes/pkg/api/v1#Handler) implemented by
+the Container. There are three types of handlers:
+
+* [ExecAction](/docs/resources-reference/v1.5/#execaction-v1):
+  Executes a specified command inside the Container. The diagnostic
+  is considered successful if the command exits with a status code of 0.
+
+* [TCPSocketAction](/docs/resources-reference/v1.5/#tcpsocketaction-v1):
+  Performs a TCP check against the Container's IP address on
+  a specified port. The diagnostic is considered successful if the port is open.
+
+* [HTTPGetAction](/docs/resources-reference/v1.5/#httpgetaction-v1):
+  Performs an HTTP Get request against the Container's IP
+  address on a specified port and path. The diagnostic is considered successful
+  if the response has a status code greater than or equal to 200 and less than 400.
+
+Each probe has one of three results:
+
+* Success: The Container passed the diagnostic.
+* Failure: The Container failed the diagnostic.
+* Unknown: The diagnostic failed, so no action should be taken.
```

Comment:
@pwittrock, What could we say here? Could we say, "You need to redesign/debug your handler"?